hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/client/QuorumJournalManager.java

```diff
@@ -518,10 +517,7 @@ public void selectInputStreams(Collection<EditLogInputStream> streams,
     // the cache used for RPC calls is not enabled; fall back to using the
     // streaming mechanism to serve such requests
     if (inProgressOk && inProgressTailingEnabled) {
-      if (LOG.isDebugEnabled()) {
```

Comment:
I have one question to ask, that is why some code should remove LOG.isDebugEnabled()?

Reply:
Thanks @slfan1989 for your review. 

Removing `LOG.isDebugEnabled()` can save a redundant judgment. 
Please refer to https://www.slf4j.org/faq.html#logging_performance to get some detailed information. Thanks